lib/cocoapods/installer/xcode/script_phase_input_output_paths.rb

```diff
@@ -0,0 +1,40 @@
+module Pod
+  class Installer
+    class Xcode
+      module ScriptPhaseInputOutputPaths
+        module_function
+
+        def use_xcfilelist?(project)
+          project.object_version.to_i >= 50
+        end
+
+        def update_script_phase_paths(script_phase, file_list_directory, input_paths: [], output_paths: [])
+          if use_xcfilelist?(script_phase.project)
+            script_phase.input_paths &&= nil
+            script_phase.output_paths &&= nil
+
+            input_file_list_path = file_list_directory.join('input_files.xcfilelist')
+            PodsProjectGenerator::TargetInstallerHelper.update_changed_file(Generator::Constant.new(input_paths.join("\n")), input_file_list_path)
+            script_phase.input_file_list_paths = [input_file_list_path.to_s]
```

Comment:
this needs to not put an absolute path into the Xcode project...

Reply:
does it? didn't realize since I didn't see any in https://github.com/CocoaPods/cocoapods-integration-specs/pull/187

Reply:
its because the integration specs dont actually have input & output paths in them

Reply:
oh right, that makes sense